src/Components/Avatar/Avatar.js

```diff
@@ -0,0 +1,72 @@
+
+import React from 'react';
+import 'antd/dist/antd.css';
+import { Upload, Icon, message } from 'antd';
+
+function getBase64(img, callback) {
```

Comment:
Why this functions are not inside the classes? Maybe the component could have a utils

Reply:
I really like this idea!